lib/Platform.Posix.cs

```diff
@@ -18,9 +18,13 @@ public static class Posix
 
```

Comment:
The file should probably be removed (it contains several merge conflict markers)

Reply:
Yes, I don't know why this old commit was taken into account... I removed this file in the other commits.

Reply:
Hm, no... could you check again? As far as I can see the file is still there.

Reply:
Oh, yes... You can safely remove this file, ZeroMQ.*.csproj doesn't use this file anymore.